authors/apps/articles/models.py

```diff
@@ -1,27 +1,33 @@
 from django.db import models
+
 from authors.apps.authentication.models import User
+from authors.apps.profiles.models import Profile
 
 # Create your models here.
 
 
 class Articles(models.Model):
     """ Model for all articles """
-    slug = models.SlugField(max_length=250, default='non')
-    title = models.CharField(max_length=50, default='non')
-    description = models.CharField(max_length=250, default='non')
-    body = models.CharField(max_length=550, default='non')
-    tagList = models.CharField(max_length=50,
-                               default='non')  # ["dragons", "training"],
+    slug = models.SlugField(unique=True, max_length=250)
+    title = models.CharField(max_length=250)
+    description = models.CharField(max_length=350)
+    body = models.TextField()
+    tagList = models.CharField(max_length=200)
     createdAt = models.DateTimeField(auto_now_add=True)
     updatedAt = models.DateTimeField(auto_now=True)
     favorited = models.BooleanField(default=False)
     favoritesCount = models.IntegerField(default=0)
     readtime = models.IntegerField(default=0)
-    author = models.ForeignKey(User, default=0, on_delete=models.CASCADE)
-
-    def __str__(self):
-        """ String representation of db object """
-        return ' {}: {}'.format(self.id, self.slug)
+    author = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    read_count = models.IntegerField(default=0)
 
     class Meta:
         ordering = ["-createdAt", "-updatedAt"]
+
+
+class ArticlesRead(models.Model):
+    """
+    model for articles read
+    """
+    user = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    slug = models.TextField()
```

Comment:
While the `Textfield` data type is fine, for consistency, I think it would be better if the slug's data type matched the slug in the Articles model.